src/ol3cesium.js

```diff
@@ -11,12 +11,26 @@ goog.require('olcs.VectorSynchronizer');
 
 
 /**
- * @param {!ol.Map} map
+ * @param {!(ol.Map|olcsx.OLCesiumOptions)} arg1 `ol.Map` or options
  * @param {Element|string=} opt_target Target element for the Cesium container.
+ *   Overrides a target set in options.
  * @constructor
  * @api
  */
-olcs.OLCesium = function(map, opt_target) {
+olcs.OLCesium = function(arg1, opt_target) {
```

Comment:
Keep in mind that if you are really attached to a particular version number you can always go with [Sentimental Versioning](http://sentimentalversioning.org/) instead.  That would let you pick your favorite version numbers and break things without feeling bad.